src/proto/api_messages_test.js

```diff
@@ -93,6 +107,10 @@ describe('api_messages', () => {
       analyticscontext.setLabelList([]);
       analyticsrequest.setContext(analyticscontext);
       analyticsrequest.setEvent(AnalyticsEvent.UNKNOWN);
+      const /** !AnalyticsEventMeta  */ analyticseventmeta = new AnalyticsEventMeta();
+      analyticseventmeta.setEventOriginator(EventOriginator.UNKNOWN_CLIENT);
```

Comment:
is the indent right for this? looks wierd here but may be different in your IDE etc

Reply:
I think I've fixed the indenting